docs/source/conf.py

```diff
@@ -138,7 +138,8 @@ def run(self):
 
 html_js_files = [
 	('https://cdn.jsdelivr.net/npm/@docsearch/js@3', {'defer': 'defer'}),
-	('js/algolia.js', {'defer': 'defer'})
+	('js/algolia.js', {'defer': 'defer'}),
+	('js/readthedoc-flyout.js', {'defer': 'defer'})
```

Comment:
you can add a trailing comma here, so that next time when adding or deleting items near the end of the list, we don't need to modify unrelated lines